azkaban-web-server/src/test/java/azkaban/webapp/servlet/LoginAbstractAzkabanServletTest.java

```diff
@@ -17,22 +17,46 @@
 package azkaban.webapp.servlet;
 
 
+import static azkaban.ServiceProvider.SERVICE_PROVIDER;
 import static junit.framework.TestCase.assertEquals;
 import static junit.framework.TestCase.assertNotSame;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+import azkaban.AzkabanCommonModule;
 import azkaban.fixture.MockLoginAzkabanServlet;
+import azkaban.utils.Props;
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
 import org.junit.Test;
 
 public class LoginAbstractAzkabanServletTest {
 
+  @Before
+  public void setUp() {
+    // TODO: reallocf remove setUp() when LoginAbstractServlet fully guicified
+    final Props props = new Props();
```

Comment:
I can do that. Do you think it's a good idea to build that infrastructure when this code will (hopefully) be retired quickly?